airbyte-integrations/connectors/destination-redis/src/main/java/io/airbyte/integrations/destination/redis/RedisDestination.java

```diff
@@ -35,21 +36,16 @@ public static Destination sshWrappedDestination() {
 
   @Override
   public AirbyteConnectionStatus check(JsonNode config) {
-    var redisConfig = new RedisConfig(config);
-
-    RedisCache redisCache = null;
-    try {
-      redisCache = RedisCacheFactory.newInstance(redisConfig);
+    try (RedisCache redisCache = RedisCacheFactory.newInstance(config)) {
```

Comment:
great usage of `try-with` block and removing extraneous code for better code readability